test_igor.py

```diff
@@ -27,26 +28,199 @@ def test_incorrect_login(self):
         login = driver.find_element(by=By.XPATH, value='.//input[@placeholder="Pick a username"]')
         random_loging = ''.join(random.choices(string.ascii_lowercase + string.digits, k=10))
         login.send_keys(random_loging)
-        sleep(3)
+        sleep(1)
 
         # Fill Email
         email = driver.find_element(by=By.XPATH, value='.//input[@placeholder="you@example.com"]')
         random_email = random_loging + "@gmail.com"
         email.send_keys(random_email)
-        sleep(3)
+        sleep(1)
 
         # Fill Password
         password = driver.find_element(by=By.XPATH, value='.//input[@placeholder="Create a password"]')
         random_password = ''.join(random.choices(string.ascii_lowercase + string.digits, k=12))
         password.send_keys(random_password)
-        sleep(3)
+        sleep(1)
 
         # Click button
         button = driver.find_element(by=By.XPATH, value=".//button[@type='submit']")
         button.click()
-        sleep(3)
+        sleep(1)
+
+        # Successful registration check
+        success_reg = driver.find_element(by=By.XPATH, value=".//span[@class='text-white mr-2']")
+        assert success_reg.text == f"{random_loging}", f"Actual message: {success_reg.text}"
+        sleep(1)
+
+    #
+    def test_2_registration_2(self):
```

Comment:
Is there any better naming then test_2_registration_2 ? Something based on test purpose ?)

Reply:
I will rename)